webview/src/experiments/components/table/RowContextMenu.tsx

```diff
@@ -202,6 +212,20 @@ const getSingleSelectMenuOptions = (
       isWorkspace,
       !hasRunningExperiment
     ),
+    experimentMenuOption(
+      [id],
+      'Plot',
+      MessageFromWebviewType.SET_EXPERIMENTS_AND_OPEN_PLOTS,
+      false,
+      false
```

Comment:
[I] The UX here is a bit broken. If I already have experiments selected and I select 'plot' for a single experiment I would expect for the action to add this experiment to the existing selections not replace them:

https://user-images.githubusercontent.com/37993418/197085608-6efac9c6-2736-4e40-ac9a-f9737bf4fed5.mov

This will also mean that if there are already 7 experiments selected that the option should be greyed out/unavailable.